showcase/src/app/demo/table/demo-table.component.ts

```diff
@@ -32,8 +32,14 @@ export class DemoTableComponent implements OnInit {
 		return 0;
 	};
 
+	public getRowClassName = (_row: any, index: number): Object => (index % 2 === 0 ? "even" : "odd");
```

Comment:
We should use the `object` type instead of `Object`. Why? https://www.typescriptlang.org/docs/handbook/declaration-files/do-s-and-don-ts.html

Reply:
Even worse, it should be string. 😆